src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,9 +7,12 @@ public class CliSyntax {
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
+    public static final Prefix PREFIX_CLASS_GROUP = new Prefix("c/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_TELEGRAM = new Prefix("t/");
+    public static final Prefix PREFIX_GITHUB = new Prefix("g/");
+    public static final Prefix PREFIX_TAG = new Prefix("tg/");
```

Comment:
Maybe we can also delete references to tags/

Reply:
Good catch Axel - I'll remove it from some common files now, but I suspect we'll find more references as we work on other commands